docs/user-guide/install.md

```diff
@@ -51,6 +51,28 @@ docker run -d \
 
 ### Unraid Docker
 
+ 1. Install the Commuinity Applications plugin by pasting the following URL in the Plugins / Install Plugin section of UnRAID  
+
+```
+https://raw.githubusercontent.com/Squidly271/community.applications/master/plugins/community.applications.plg
+```  
+
+ ![Community Applications Install](../images/unraid-install-community-apps.png)  
+2. Click on the `Apps` tab in UnRAID, search for `ersatztv`, and click the `install` button.
+
+![Search Community Applications ErsatzTV](../images/unraid-docker-install-ersatz.png)
+3. Choose an appropriate branch (Use `vaapi` for video acceleration for AMD GPUs and Intel CPUs with QuickSync, use `nvidia` for video acceleration for nVidia GPUs)
```

Comment:
Removing the line break between the image link and the next steps renders the step numbers properly.